frontend/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+  id "com.moowork.node" version "1.2.0"
+}
+
+node {
+  version = '10.15.0'
+  npmVersion = '6.4.1'
```

Comment:
I think nmp 6.6.0 is the current one. Could you please update? Thank you :)

Reply:
Sure, but just to check, after running `npm i -g npm@latest` the version is 6.7.0
Is 6.6.0 a LTS version? I can't find much information on npm versions, only [this](https://nodejs.org/en/download/releases/)

Reply:
Yes, the latest is 6.7.0, released apparantely yesterday ;)